test/e2e/storage/persistent_volumes-local.go

```diff
@@ -504,6 +537,32 @@ func makeLocalPod(config *localTestConfig, volume *localTestVolume, cmd string)
 	return framework.MakeSecPod(config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, cmd, false, false, selinuxLabel)
 }
 
+func makeLocalPodWithNodeSelector(config *localTestConfig, volume *localTestVolume, nodeSelector string) (pod *v1.Pod) {
```

Comment:
The string is actually a nodeName, not a nodeSelector.

Reply:
will change it.